www/docs/docs/custom-layouts.md

```diff
@@ -0,0 +1,60 @@
+---
+---
+
+# Custom Layouts
+
+## Defining Custom Layouts
+
+`react-ts-form` supports custom layouts for times when we need more than props just props to achieve the desired style (for example when we want two inputs in the same row).
+
+To build a form with a custom layout, pass a function that returns JSX as the child to your form component, and that function will be passed the form elements as a prop:
```

Comment:
Docs look great! Thank you for taking this!

Reply:
Thanks for your awesome solution! 🙏